virtual_rainforest/models/animals/animal_communities.py

```diff
@@ -41,6 +44,9 @@ def __init__(
             k.name: [] for k in self.functional_groups
         }
         """Generate a dictionary of functional groups within the community."""
+        self.plant_community: PlantCommunity = PlantCommunity(10000.0, 1)
+        self.carcass_pool: CarcassPool = CarcassPool(10000.0, 1)
+        self.soil_pool: PalatableSoil = PalatableSoil(10000.0, 1)
```

Comment:
Why 10000.0 specifically?

Reply:
It's a completely toy value that was round enough to be easy and big enough to not always be going to zero from a trophic interaction.